.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+randomHexColor.js
+draft.js
+colorNameToHex.js
+yarn.error.lock
```

Comment:
I think this is supposed to be something slightly different - maybe with `-` instead of period? Oh and `log` instead of `lock`?